style.css

```diff
@@ -500,3 +501,143 @@ header {
   background: #fff;
   color: black;
 }
+
+@media (min-width: 764px) {
+  .vector {
+    display: none;
+  }
+
+  .main {
+    width: 98.9vw;
+    background-image: url('assets/desktop\ files/header.png');
+    padding: 200px 24px;
+    height: 800px;
+  }
+
+  .logo {
+    padding: 10px 12px;
+    margin-left: 143px;
+  }
+
+  .nav-list {
+    display: flex;
+    justify-content: space-between;
+    margin-right: 143px;
+    align-items: flex-start;
+  }
+
+  .nav-list li {
+    list-style-type: none;
+    padding: 10px 12px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .texts {
+    margin-left: 15%;
+    width: 60%;
+  }
+
+  .container {
+    width: 96%;
+  }
+
+  .works {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    height: 496px;
+    padding: 24px;
+  }
+
+  .snapshot {
+    width: 46%;
+    height: 448px;
+  }
+
+  .snapshot img {
+    width: 100%;
+    height: 100%;
+  }
+
+  .info {
+    padding: 24px;
+    width: 48%;
+    height: 448px;
+  }
+
+  .desc {
+    width: 100%;
+  }
+
+  .projects {
+    flex-direction: row-reverse;
+  }
+
+  .about {
+    display: flex;
+    flex-direction: column;
+    width: 98.9vw;
+    justify-content: center;
+    border: 1px solid gray;
+  }
+
+  .about-elements {
+    display: flex;
+  }
+
+  .first-element {
+    width: 35%;
+    margin-left: 10%;
+    margin-right: 20px;
+  }
+
+  .skills {
+    width: 50%;
+  }
+
+  .content {
+    width: 100%;
+    height: 200px;
+  }
+
+  .technology-list {
+    width: 100%;
+  }
+
+  .line {
+    width: 100%;
+  }
+
+  .techs {
+    display: flex;
+    flex-direction: row;
+    gap: 10px;
+  }
+
+  .item {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    width: 120px;
+    height: 100%;
+  }
+
+  .tech-name {
+    margin-left: 1px;
+  }
+
+  .contact-form {
+    width: 98.9vw;
+    background-image: url("assets/desktop\ files/contact form-back.png");
+    background-repeat: no-repeat;
+  }
```

Comment:
- [ ] I noticed a grey patch at the bottom part of your design on both the `mobile` and `desktop` versions. This does not conform with the design on Figma, kindly change it. 👍 

### Your Design

![Screenshot 2023-01-26 220636](https://user-images.githubusercontent.com/77106962/214950577-ef4b4710-ed6b-43a5-be31-0551bbd65825.png)

![mobile](https://user-images.githubusercontent.com/77106962/214950978-3744c2d7-fbab-4623-aed0-9ae3915fd051.png)


### Figma Design

![image](https://user-images.githubusercontent.com/77106962/214950645-cc493e11-bbae-4ca0-b2a6-494ccda8899d.png)

![image](https://user-images.githubusercontent.com/77106962/214951086-9a56e108-01fd-4f74-8010-898659f7e58c.png)